### PR TITLE
Generator check in

### PR DESCRIPTION
**Pull request by ryanthemanuel**

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Cerner Corporation
+
+- {First Name} {Last Name} [@{github username}]
+
+[@{github username}]: https://github.com/{github username}
```
Comment (Matt-Butler) [inline CONTRIBUTORS.md]: maybe add your name now so we dont' forget

Comment (ryanthemanuel) [inline CONTRIBUTORS.md]: Good call.  Done.

**Comment by Matt-Butler** [comment]
+1

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Cerner Corporation
+
+- Ryan Manuel [@ryanthemanuel}]
```
Comment (Matt-Butler) [inline CONTRIBUTORS.md]: https://cl.ly/1y2C3d183j2b

[@ryanthemanuel}] => [@ryanthemanuel]

Comment (ryanthemanuel) [inline CONTRIBUTORS.md]: Good catch.  Fixed.

**Pull request merged**